src/assets/locales/de/main.json

```diff
@@ -1307,7 +1307,7 @@
       }
     },
     "serviceMarketplace": {
-      "headline": "Purchase Service Request",
+      "headline": "Service-Abonnement",
```

Comment:
@lavanya-bmw  is it correct that the PR includes this change?

Reply:
@jjeroch this was not part of this ticket but added this here since it's a small heading change

Reply:
@jjeroch Please confirm If I need to raise a new PR for this change?

Reply:
we can proceed with the same